### ISSUE TITLE
Unable to build the project

### ISSUE THREAD

**Issue by kschulst** [GreaterMKEMeetup/spring-restdocs-postman#4]
`gradle build` fails with `> Could not get unknown property 'ossrhUsername' for root project 'spring-restdocs-postman' of type org.gradle.api.Project.
`
(used by the uploadArchives task).

Could this configuration be made optional?

**Comment by aglassman** [comment]
Yes, there is probably a way to avoid having the signing and uploading task run by default.  I'll look into configuring the build for those that wont be signing, and uploading artifacts.  Thanks!
